startup_heroku.sh

```diff
@@ -38,7 +38,11 @@ echo "*NOTE.* You may have to refresh as the app can be slow to start"
 echo
 export HEROKU_URL=`heroku apps:info --app $APP_NAME | grep "Web URL" | cut -c16-`
 export SMART_ANSWER_TO_OPEN="marriage-abroad"
-open $HEROKU_URL$SMART_ANSWER_TO_OPEN
+if type open 2>/dev/null; then
+    open $HEROKU_URL$SMART_ANSWER_TO_OPEN
+else
+    echo $HEROKU_URL$SMART_ANSWER_TO_OPEN
```

Comment:
Can we also have a message saying we didn't open the page and why? As the previous echo statement says that we're doing it. If nothing happens it might seem that something is broken.